301-subnet-driven-deployment/WinServ.json

```diff
@@ -0,0 +1,403 @@
+        "description": "Computer Name"
+      }
+    },
+    "domainJoinPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The password for to join the domain"
+      }
+    },
+    "domainJoinUsername": {
+      "type": "string",
+      "defaultValue": "zeAdmin",
+      "metadata": {
+        "description": "The user for to join the domain"
+      }
+    },
+    "domainName": {
+      "type": "string",
+      "defaultValue": "fabrikam.com",
+      "metadata": {
+        "description": "The FQDN of the AD Domain created "
+      }
+    },
+    "imageOffer": {
+      "type": "string",
+      "defaultValue": "WindowsServer",
+      "metadata": {
+        "description": "Image Offer"
+      }
+    },
+    "imagePublisher": {
+      "type": "string",
+      "defaultValue": "MicrosoftWindowsServer",
+      "metadata": {
+        "description": "Image Publisher"
+      }
+    },
+    "imageSKU": {
+      "type": "string",
+      "defaultValue": "2012-R2-Datacenter",
+      "metadata": {
+        "description": "Image SKU"
+      }
+    },
+    "imageVersion": {
+      "type": "string",
+      "defaultValue": "latest",
+      "metadata": {
+        "description": "Image version, 'latest' by default (recommended)"
+      }
+
+    },
+    "indx": {
+      "type": "int",
+      "defaultValue": 0,
+      "metadata": {
+        "description": "suffix to avoid collisions"
+      }
+    },
+    "lbName": {
+      "type": "string",
+      "defaultValue": "lbn",
+      "metadata": {
+        "description": "Front End load Balancer Name'."
+      }
+    },
+    "natRulePrefix": {
+      "type": "string",
+      "defaultValue": "nr-",
+      "metadata": {
+        "description": "Prefix for Nat Rules."
+      }
+    },
+    "nicNamePrefix": {
+      "type": "string",
+      "defaultValue": "nic-",
+      "metadata": {
+        "description": "Prefix for Nics."
+      }
+    },
+    "publicIPAddressName": {
+      "type": "string",
+      "defaultValue": "PublicIP",
+      "metadata": {
+        "description": "Public address name, for lb configuration."
+      }
+    },
+    "publicStartRdpPort": {
+      "type": "int",
+      "defaultValue": 5500,
+      "metadata": {
+        "description": "Public port, for RDP remote connection. Will be incremented if the templae is in a loop"
+      }
+    },
+    "storageAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "Storage to put the disk image on"
+      }
+    },
+    "subnets": {
+      "type": "array",
+      "metadata": {
+        "description": "list of subnets where the VM will belong to"
+      }
+    },
+    "vmNamePrefix": {
+      "type": "string",
+      "defaultValue": "srv-"
+    },
+    "vmSize": {
+      "type": "string",
+      "defaultValue": "Standard_A2",
+      "allowedValues": [ 
+        "Standard_A0",
+        "Standard_A1",
+        "Standard_A2",
+        "Standard_A3",
+        "Standard_A4",
+        "A10",
```

Comment:
Oddly, the commandlet gives these values without the prefix. Added the prefix and tried with standard_A5. This works.